app/models/csv_export.rb

```diff
@@ -17,7 +17,7 @@ class CsvExport < ActiveRecord::Base
   }
 
   def status?(state)
-    'ready' == state.to_s ? ['downloaded', 'finished'].include?(status) : status == state.to_s
+    state.to_s == 'ready' ? ['downloaded', 'finished'].include?(status) : status == state.to_s
```

Comment:
What was wrong here? why does the order matter?

Reply:
basic code style of having the variable on the right `a == 1` and not `1 == a`